fe/fe-core/src/main/java/com/starrocks/catalog/StarOSAgent.java

```diff
@@ -58,8 +61,14 @@ public StarOSAgent() {
         rwLock = new ReentrantReadWriteLock();
     }
 
+    private void prepare() {
```

Comment:
service id should be protected by lock, all other map should also be protected by lock.